tests/helpers/setup.ts

```diff
@@ -1,18 +1,18 @@
 import * as enzyme from 'enzyme';
-import * as Adapter from 'enzyme-adapter-react-15';
+import * as Adapter from 'enzyme-adapter-react-16';
 
 enzyme.configure({ adapter: new Adapter() });
 
 // This monkey patches document.createEvent so that MouseEvents have a `pageX` value.
 // This is intended to prevent React from adding window listeners when a component is mounted using enzyme.
-const originalCreateEvent = document.createEvent;
+// const originalCreateEvent = document.createEvent;
```

Comment:
Did you mean to leave the code in this file commented out?

Reply:
_I_ didn't. 😉 Will remove.